storage/object/reference-content/ssl-tls-certificate.mdx

```diff
@@ -30,26 +30,33 @@ When you enable Edge Services, initially your bucket content is served through t
 
-Therefore, when you customize your Edge Services endpoint with a subdomain, you are prompted to upload an SSL/TLS certificate for that subdomain. This certificate is stored in [Scaleway Secret Manager](/identity-and-access-management/secret-manager/quickstart/).
+Therefore, when you customize your Edge Services endpoint with a subdomain, you are prompted to generate or upload an SSL/TLS certificate for that subdomain.
 
-## How to get a certificate
+## How can I provide an SSL/TLS certificate for my Edge Services customized domain?
 
-In general, SSL/TLS certificates can either be **self-signed** (signed by the subject of the certificate, e.g. the owner of the domain) or **CA-signed** (signed by a third party **C**ertificate **A**uthority which is publicly trusted). 
+You will be prompted to choose one of the following options when [customizing your domain](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain):
 
-**Self-signed certificates cannot be used with Edge Services, all certificates must be signed by a CA.**
+- **Generate a Let's Encrypt certificate**: Scaleway generates a free, managed Let's Encrypt certificate for your domain and automatically renews it as necessary.
 
-To get an SSL/TLS certificate for your domain or subdomain, you need to generate a Certificate Signing Request (CSR) and submit it to a Certificate Authority (CA) for them to validate your domain, who then send you a signed certificate. You may be able to carry out this procedure via your hosting provider, or from the command line.
+- **Select an existing certificate from Secret Manager**: You select a certificate that you have already uploaded in [Scaleway Secret Manager](/identity-and-access-management/secret-manager/quickstart/).
+
+- **Manually import a certificate into Secret Manager**: You can manually create your own certificate and import it. It will be stored in Scaleway Secret Manager (check the [dedicated pricing page](https://www.scaleway.com/en/pricing/?tags=securityandidentity)).
+
+## Generating a managed Let's Encrypt certificate
 
-Find below some useful guides from different CAs who provide free certificates, on how to obtain a certificate from them:
+This is the hassle-free option if you do not want to create or manage your own SSL/TLS certificate. Scaleway takes care of generating a certificate for your customized domain in the correct format. The certificate is automatically renewed before it expires. This option is available for free: it costs you nothing for Scaleway to generate and manage a Let's Encrypt certificate for your domain.
```

Comment:
@RoRoJ maybe we should mention that the CNAME must be correctly set for the LE certificate option to be available in the console and then for the cert to be properly generated and renewed.